2-mandatory/4-people-i-know.js

```diff
@@ -394,6 +402,13 @@ This time, I only want the full names ("<firstname> <lastname>") of my friends w
 */
 
 let friendsWhoAreColleaguesOfStacie = [];
+
+friends.forEach((friends) =>{
+  friends.colleagues.map(friend =>{
```

Comment:
probably better to ise forEach() here. map() returns a new Array, but we don't need to do so